resolwe/flow/utils/test.py

```diff
@@ -224,6 +224,8 @@ def tearDown(self):
                 shutil.rmtree(data_dir, ignore_errors=True)
                 shutil.rmtree(export_dir, ignore_errors=True)
 
+        Process.objects.all().delete()
```

Comment:
This is interesting. All but the first test in `resolwe.flow.tests.test_executors.ManagerRunProcessTest` have failed after the update to Django 1.10 (see stacktrace below). I thought maybe processes were not removed because we register them by calling a Django command. 

```
ERROR: test_workflow (resolwe.flow.tests.test_executors.ManagerRunProcessTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/miha/.virtualenvs/resolwe/lib/python2.7/site-packages/django/test/testcases.py", line 216, in __call__
    self._post_teardown()
  File "/Users/miha/.virtualenvs/resolwe/lib/python2.7/site-packages/django/test/testcases.py", line 908, in _post_teardown
    self._fixture_teardown()
  File "/Users/miha/.virtualenvs/resolwe/lib/python2.7/site-packages/django/test/testcases.py", line 1064, in _fixture_teardown
    connections[db_name].check_constraints()
  File "/Users/miha/.virtualenvs/resolwe/lib/python2.7/site-packages/django/db/backends/postgresql/base.py", line 224, in check_constraints
    self.cursor().execute('SET CONSTRAINTS ALL IMMEDIATE')
  File "/Users/miha/.virtualenvs/resolwe/lib/python2.7/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/Users/miha/.virtualenvs/resolwe/lib/python2.7/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/Users/miha/.virtualenvs/resolwe/lib/python2.7/site-packages/django/db/backends/utils.py", line 62, in execute
    return self.cursor.execute(sql)
IntegrityError: insert or update on table "flow_process" violates foreign key constraint "flow_process_contributor_id_ae49ec6b_fk_auth_user_id"
DETAIL:  Key (contributor_id)=(30) is not present in table "auth_user".
```